opba-protocols/hbci-protocol/src/main/java/de/adorsys/opba/protocol/hbci/service/consent/HbciReadTransactionListFromCache.java

```diff
@@ -28,6 +28,10 @@ private void convertConsentToResponseIfPresent(DelegateExecution execution) {
             if (null != ctx.getCachedResult() && null != ctx.getCachedResult().getTransactionsByIban()) {
                 ctx.setResponse(ctx.getCachedResult().getTransactionsByIban().get(ctx.getAccountIban()));
             }
+
+            if (ctx.getUseCache() == Boolean.FALSE) {
```

Comment:
This will cause the accounts cache to be removed, you need only update transactions